src/backend/renderer.js

```diff
@@ -1334,6 +1334,12 @@ export function attach(
       // Does the current renderer support editable function props?
       canEditFunctionProps: typeof overrideProps === 'function',
 
+      // Is this Suspense, and can we toggle it?
+      // TODO: this will incorrectly enable it on old versions.
+      canToggleSuspense:
+        typeof overrideProps === 'function' &&
+        tag === ReactTypeOfWork.SuspenseComponent,
```

Comment:
Ah, nice TODO. This should probably look for a new injected flag that signals whether the current version _and build_ supports the override behavior.